qutip/core/qobj.py

```diff
@@ -1802,11 +1812,3 @@ def ptrace(Q, sel):
     if not isinstance(Q, Qobj):
         raise TypeError("Input is not a quantum object")
     return Q.ptrace(sel)
-
-
-# TRAILING IMPORTS
-# We do a few imports here to avoid circular dependencies.
-from qutip.core.superop_reps import to_choi
-from qutip.core.superoperator import vector_to_operator, operator_to_vector
-from qutip.core.tensor import tensor_swap, tensor
-from qutip.core import metrics as mts
```

Comment:
We can probably tell CodeClimate to be quiet if needed. :)

Separating things to avoid the circular dependencies would be awesome, but +1 on moving the imports to where they're used.